### ISSUE TITLE
Various Tweaks

### ISSUE THREAD

**Issue by jpotvin** [Xalgorithms/rm-components#5]
1. The subtitles "Character of this Obligation" etc, should be one font size larger and bold.
2. Adjust the "Qualitative Weights" subtitle to "Output: Qualitative Weights"
3. Remove "There are" from each label of the 2nd group of Qualitative Weights. 
4. Move "Select the rule category that most applies." to an "Output: Purpose" Section immediately before "Output: Qualitative Weights"
5. Adjust each of the "Input" subtitles to have a colon like:  "Input: Contexts" 
6. I'm considering a whole section for "Output" fields and a whole section of "Input" fields. But for clarity it doesn't hurt to remind the author with word repetition which aspect they are working on. So we can leave it as is I think. 
7. For "Involved Party" please line-break to drop "Standard Industry Code" beneath.
8. For "Involved Product or Service" also line-break to drop "Standard Product or Service Code" beneath (note, do not repeat "Standard Industry Code"); also change "Add Involved Party" to "Add Involved Product or Service"
9. Reminder to put the correct choices into the <grin>No Peace; No Justice</grin> lists.
10. Move the "Save Rule Information" button to be quite visible inside the left-side panel, to be hit anytime for interim saving.

...that's enough for one list of tweaks. I'll put others into a separate entry.

**Comment by millenniallglyphs** [comment]
I added in all the dropdown fields today and was able to pull all the needed fields from drafts. 
@jpotvin you suggest:
> Adjust the "Qualitative Weights" subtitle to "Output: Qualitative Weights"
Throughout the rule-maker a lot of work is needed on the copy. Our labels must be jargon-free and direct user action. Writing plays a big role in the UX of a tool. I found section two of [this article](https://uxplanet.org/ux-writing-how-to-do-it-like-google-with-this-powerful-checklist-e263cc37f5f1) did a great job of explaining how to approach this. We can work together on this, since getting this right is going to make a big difference in adoption/ease of use. 
@RyanFleck I'm happy to take on all of these. Thanks for compiling the checklist.
